dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -171,3 +171,6 @@ throwExceptionWhenUnableToCompleteOrDispatch.desc=Throw an illegal state excepti
 
 getRealPathReturnsQualifiedPath.name=Always return a non-null path from getRealPath
 getRealPathReturnsQualifiedPath.desc=Return non-null paths from the ServletContext.getRealPath(String) method, even if no resource exists at the given path. The default is true. If applications expect the getRealPath method to return null when given a path for which no resource exists, set the property to false.
+
+stopAppStartUponListenerException.name=Stop the web application when context listener fails to start due to an unhandled exception.
```

Comment:
Change,
"Stop the web application when context listener fails to start due to an unhandled exception."
to,
"Stop the web application when context listener doesn't start due to an unhandled exception."